tests/suites/test_suite_psa_crypto_se_driver_hal.function

```diff
@@ -587,17 +588,26 @@ void key_creation_in_chosen_slot( int slot_arg,
                              &handle );
     TEST_EQUAL( status, expected_status );
 
-    if( status == PSA_SUCCESS )
+    if( status != PSA_SUCCESS )
+        goto exit;
+
+    /* Maybe restart, to check that the information is saved correctly. */
+    if( restart )
```

Comment:
I don't know if it matters, but if this passes, we won't know, if the key was fine before the restart.

Reply:
The test data generally calls this function first with `restart=0`, then again with the same parameters except for `restart=1`. So the first call checks that everything works in memory and the second call checks that the storage part works.